localization/english/religion/wc_religion_l_english.yml

```
@@ -5,4 +5,502 @@
+ 
+ helya:0 "Helism"
+ helya_adj:0 "Helic"
+ helya_adherent:0 "Helist"
+ helya_adherent_plural:0 "Helists"
+ helya_desc:0 "Helya, the matriarch of Helheim, which is the underworld of the Vrykul who have fallen and haven't ascended to the Halls of Valour, was a Watcher who assisted Odyn in imprisoning elementals into the Elemental Plane. She would be the first spectral Val'kyr when she rebelled against Odyn. She would soon create a champion who would bear Odyns Warswords of the Valajar, which Helya stole from Odyn. This champion would slay many for their souls to serve in Helheim. She would also create the Helarjar, a holy order mocking Odyn's Valarjar. The Kvaldir of Helya would also be responsible for murdering those of dishonour and unworth, taking them to Helheim."
+ 
+ pandaren_religion_group:0 "Chi"
+ pandaren_religion_group_adj:0 "Chi"
+ pandaren_religion_group_adherent:0 "Chiist"
+ pandaren_religion_group_adherent_plural:0 "Chiists"
+ pandaren_religion_group_desc:0 "The philosophy of Chi is based on two things: overcoming negative emotions and spiritual ascension. Both goals are achieved through meditation and self-control."
+ 
+ august_celestials:0 "Celestism"
+ august_celestials_adj:0 "Celestial"
+ august_celestials_adherent:0 "Celestist"
+ august_celestials_adherent_plural:0 "Celestists"
+ august_celestials_desc:0 "The August Celestials are four wild gods of Pandaria. The celestials built temples to train monks in their respective styles after the pandaren revolution. Many native races of Pandaria respect these creatures and worship them as gods. The Celestials teach to overcome negative emotions and develop a spiritual connection with the world through meditations."
+ 
+ nerub_group:0 "Nerub"
+ nerub_group_adj:0 "Nerub"
+ nerub_group_adherent:0 "Nerub"
+ nerub_group_adherent_plural:0 "Nerubs"
+ nerub_group_desc:0 "Nerub philosophies are very different from the beliefs of the Upper World. Born in the dark tunnels beneath Northrend, they preach about the importance of society and its laws."
+ 
+ spider_philosophy:0 "Nazen"
+ spider_philosophy_adj:0 "Nazeni"
+ spider_philosophy_adherent:0 "Nazeni"
+ spider_philosophy_adherent_plural:0 "Nazeni"
+ spider_philosophy_desc:0 "$spider_philosophy_adj$ philosophy preaches renunciation of emotions. The life of a particular individual is not important when it comes to society. Thus $spider_philosophy_adherent_plural$ should always be willing to sacrifice themselves for the greater good."
+ 
+ secular_group:0 "Secularism"
+ secular_group_adj:0 "Secular"
+ secular_group_adherent:0 "Secularist"
+ secular_group_adherent_plural:0 "Secularists"
+ secular_group_desc:0 "A broad group of secular philosophies. $secular_group_adherent_plural$ don't worship any god since they don't see the point in this."
```

Made a slight change. I'd advise rehashing this though. Secular societies aren't atheists, which this is implying, they just are neutral in terms of religion, and accept free worship.
```suggestion
 secular_group_desc:0 "A broad group of secular philosophies. $secular_group_adherent_plural$ don't worship any god since they don't see the point in religion."
```